realm/realm-library/src/main/java/io/realm/MutableRealmSchema.java

```diff
@@ -42,6 +44,20 @@ public RealmObjectSchema get(String className) {
         return new MutableRealmObjectSchema(realm, this, table);
     }
 
+    @Override
+    public Set<RealmObjectSchema> getAll() {
+        // Return all tables prefixed with class__ in the Realm file
+        int tableCount = (int) realm.getSharedRealm().size();
```

Comment:
This just makes me think if we're throwing an exception when using a DynamicRealm with a RealmConfiguration containing a `module` conf, since here we're grabbing all the classes for DynamicRealm

Reply:
Since the same configuration is used by both normal and dynamic Realms, we cannot really throw when creating the configuration. Throwing when opening the dynamic Realm seems rather harsh?